scbs/MethylVI_batch.ipynb

```diff
@@ -0,0 +1,468 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        os.system(f"wget -q -O {mdata_path} https://figshare.com/ndownloader/files/49632108")
</pre><p>@ori can we use our download function instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scvi-tutorials/pull/363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
not sure, instead of what?

Reply:
replaces wget with pooch